docs/patterns/navigation.md

```diff
@@ -34,6 +36,18 @@ by adjusting the `$breakpoint-navigation-threshold` in `_settings_breakpoints.sc
 View example of the navigation pattern
 </a>
 
+<span class="p-label--deprecated">Deprecated</span>
+
+In Vanilla 2.8 we deprecated the use of `p-navigation__links`, `p-navigation__link` and classless `<a>` in the navigation. Support for these classes will be removed in future version 3.0.
+
+You should use `p-navigation__items`, `p-navigation__item` and `<a class="p-navigation__link">` instead.
```

Comment:
Maybe we should start a 3.0 upgrade document now so its easier when it comes to it?

Reply:
Created issue about it #2858